logger_test.go

```diff
@@ -0,0 +1,194 @@
+package logger_test
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	log "github.com/g2a-com/klio-logger-go"
+)
+
+func TestParseLevel(t *testing.T) {
+	var l log.Level
+	var ok bool
+
+	l, ok = log.ParseLevel("spam")
```

Comment:
You could create subtests for each level: ```t.Run("it should parse spam", func (t *testing.T) { … })```.
It is basically easier to determine which test fails if you have just few asserts per test ;)